pkg/appStore/deployment/fullMode/AppStoreDeploymentFullModeService.go

```diff
@@ -193,27 +199,27 @@ func (impl AppStoreDeploymentFullModeServiceImpl) AppStoreDeployOperationGIT(ins
 	valuesByte, err := json.Marshal(valuesMap)
 	if err != nil {
 		impl.logger.Errorw("error in marshaling", "err", err)
-		return installAppVersionRequest, nil, err
+		return nil, nil, err
 	}
 
-	valuesYaml := &util.ChartConfig{
+	valuesYamlConfig := &util.ChartConfig{
 		FileName:       appStoreBean.VALUES_YAML_FILE,
 		FileContent:    string(valuesByte),
 		ChartName:      chartMeta.Name,
 		ChartLocation:  argocdAppName,
-		ChartRepoName:  chartMeta.Name,
+		ChartRepoName:  gitOpsRepoName,
```

Comment:
I am ignoring reviewing this business logic as I don't have context of this story